index.html

```diff
@@ -1900,22 +1900,37 @@ <h2 id="utility">Utility Functions</h2>
       <p id="iteratee">
         <b class="header">iteratee</b><code>_.iteratee(value, [context])</code>
         <br />
-        A mostly-internal function to generate callbacks that can be applied
-        to each element in a collection, returning the desired result — either
-        identity, an arbitrary callback, a property matcher, or a property accessor.
-        <br />
-        The full list of Underscore methods that transform predicates
-        through <tt>_.iteratee</tt> is <tt>countBy</tt>, <tt>every</tt>, <tt>filter</tt>,
-        <tt>find</tt>, <tt>findIndex</tt>, <tt>findKey</tt>, <tt>findLastIndex</tt>,
-        <tt>groupBy</tt>, <tt>indexBy</tt>, <tt>map</tt>, <tt>mapObject</tt>, <tt>max</tt>,
-        <tt>min</tt>, <tt>partition</tt>, <tt>reject</tt>, <tt>some</tt>, <tt>sortBy</tt>,
-        <tt>sortedIndex</tt>, and <tt>uniq</tt>
+        Generates a callback that can be applied to each element in
+        a collection. <tt>_.iteratee</tt> supports a number of shorthand
+        syntaxes for common callback use cases. Depending <tt>value</tt>'s type
+        <tt>_.iteratee</tt> will return:
       </p>
       <pre>
-var stooges = [{name: 'curly', age: 25}, {name: 'moe', age: 21}, {name: 'larry', age: 23}];
-_.map(stooges, _.iteratee('age'));
-=&gt; [25, 21, 23];
-</pre>
+// No value
+_.iteratee();
+=> _.identity()
+
+// Function
+_.iteratee(function(n) { return n * 2; });
+=> function(n) { return n * 2; }
+
+// Object
+_.iteratee({firstName: 'Chelsea'});
+=> _.matcher({firstName: 'Chelsea'});
+
+// Anything else
+_.iteratee('firstName');
+=> _.property('firstName');</pre>
+
+      <p>
+        The following Underscore methods transform their predicates through
```

Comment:
Is `predicates` the right word here? I think of a predicate as a subset of iteratee that returns true or false (nb, 
https://github.com/jashkenas/underscore/pull/2399#issuecomment-170646769), as opposed to the raw param (identity, arbitrary callback, property matcher, or property accessor) that `_.iteratee` transforms. (As an aside, do we have a term for that raw param?)

How about `The following Underscore methods generate a callback through <tt>_.iteratee</tt>:`?